system/core/pages.php

```diff
@@ -4,21 +4,53 @@
 	/*
 		Title: Pages
 
-		This chapter describes the way pages are registered.
+		This chapter describes the way views and pages are registered.
 	*/
 
 	/*
-		Class: Page
-		abstract class for handling a certain kind of data
+		Class: HyphaPage
+		abstract class for a page that can process a request.
 	*/
-	abstract class Page {
-		public $pageListNode, $html, $language, $pagename, $O_O, $args, $privateFlag;
-		function __construct($node, RequestContext $O_O) {
+	abstract class HyphaPage {
+		public $html, $language, $pagename, $O_O, $args, $privateFlag;
+
+		function __construct(RequestContext $O_O) {
 			global $hyphaHtml;
```

Comment:
Curious if this global could not be passed through constructor? (as a non blocking review suggestion :-) )

Reply:
Yeah, ideally we would get rid of all globals at some point, but that would be a bigger refactor than I want to do now.

Reply:
Understood :-)